core/src/main/scala/com/salesforce/op/evaluators/EvaluationMetrics.scala

```diff
@@ -176,6 +176,22 @@ object RegressionEvalMetrics extends Enum[RegressionEvalMetric] {
 }
 
 
+/**
+ * Forecast Metrics
+ */
+sealed abstract class ForecastEvalMetric
+(
+  val sparkEntryName: String,
+  val humanFriendlyName: String
+) extends EvalMetric
+
+
+object ForecastEvalMetrics extends Enum[RegressionEvalMetric] {
+  val values: Seq[RegressionEvalMetric] = findValues
+  case object sMAPE extends RegressionEvalMetric("smape", "symmetric Mean Absolute Percentage Error")
+}
+
```

Comment:
1. please also add ForecastEvalMetrics to `EvalMetric.withNameInsensitive` function see Line 109
2. as well as add a new entry of `Forecast` to `OpEvaluatorNames`